### PR TITLE
192 check workflow constraints

### PR DESCRIPTION
**Pull request by eriktamsen**
improved constraint

### CODE REVIEW

File: lebedigital/demonstrator_scripts/dummy_paste_strength_stiffness.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def dummy_paste_strength_stiffness(slag_ratio, phi_paste):
         paste_youngs_modulus_min + (paste_youngs_modulus_max - paste_youngs_modulus_min) * slag_ratio
     )
```
Comment (atulag0711) [inline lebedigital/demonstrator_scripts/dummy_paste_strength_stiffness.py:32]: E also needs to be updated right? similar to the paste_strength i.e., 
paste_youngs_modulus = (
        paste_youngs_modulus_max - (paste_youngs_modulus_max - paste_youngs_modulus_min) * slag_ratio

File: tests/demonstrator_scripts/test_dummy_scripts.py [hunk 2]
```diff
@@ -16,4 +18,4 @@ def test_dummy_scripts():
 
     E, fc = dummy_paste_strength_stiffness(0, 10)
     assert E.magnitude == pytest.approx(30)
```
Comment (atulag0711) [inline tests/demonstrator_scripts/test_dummy_scripts.py:20]: I guess then accordingly this will change.

**Review by atulag0711 — APPROVED** [review]
Thanks Erik. Just minor changes in the dummy E script for convenience.

File: lebedigital/demonstrator_scripts/dummy_paste_strength_stiffness.py (hunk 1 above)
Comment (eriktamsen) [reply lebedigital/demonstrator_scripts/dummy_paste_strength_stiffness.py:32]: Needs is a strong word :D, this is a dummy script but by implementing the change the trend might be closer to what we would expect in real life.
I made the change as requested.

File: tests/demonstrator_scripts/test_dummy_scripts.py (hunk 2 above)
Comment (eriktamsen) [reply tests/demonstrator_scripts/test_dummy_scripts.py]: Changed accordingly

**Pull request merged**